tests/integration/unbonding.go

```diff
@@ -461,3 +461,47 @@ func (s *CCVTestSuite) TestRedelegationProviderFirst() {
 	// Check that ccv unbonding op has been deleted
 	checkCCVUnbondingOp(s, s.providerCtx(), s.consumerChain.ChainID, valsetUpdateID, false)
 }
+
+func (s *CCVTestSuite) TestTooManyLastValidators() {
+	sk := s.providerApp.GetTestStakingKeeper()
+
+	// get current staking params
+	p := sk.GetParams(s.providerCtx())
+
+	// get validators, which are all active atm
```

Comment:
nit (being pedantic): 
```suggestion
	// get validators, which are all active at the moment
```
:)